### PR TITLE
Improve clarity of definition to be less ambiguous

### PR DESCRIPTION
**Pull request by EmmaJP**
Added a change that makes it clear that a role value will come from a recognised standardised list. This is for clarity, to reduce ambiguity, with recognition that an invalid explicit role will default to an implicit role as determined by the browser.

Closes issue(s):

- https://github.com/act-rules/act-rules.github.io/issues/1129

Need for Final Call:
This will not require a Final Call as it is clarifying a definition rather than changing any meaning or rules that depend on it.

---

### **When creating PR:**

- Make sure you requesting to **pull a branch** (right side) to the `develop` branch (left side).

### **After creating PR:**

- Add yourself (and co-authors) as "Assignees" for PR.
- Add label to indicate if it's a `Rule`, `Definition` or `Chore`.
- Close the issue that the PR resolves (and make sure the issue is referenced in the top of this comment)
- Optionally request feedback from anyone in particular by assigning them as "Reviewers".

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Final Call period. In case of disagreement, the longer period wins.

### CODE REVIEW

File: pages/glossary/semantic-role.md [hunk 1]
```diff
@@ -5,6 +5,6 @@ unambiguous: true
 objective: true
 ---
 
-A _semantic [role](https://www.w3.org/TR/wai-aria-1.1/#dfn-role)_ is a semantic association that indicates an object's type. This allows tools to present and support interaction with the object in a manner that is consistent with user expectations about other objects of that type.
+A _semantic [role](https://www.w3.org/TR/wai-aria-1.1/#dfn-role)_ is a semantic association that indicates an object's type, as defined by a standard such as the [WAI-ARIA spec](https://www.w3.org/TR/wai-aria-1.1/#role_definitions), [Graphics AAM spec](https://w3c.github.io/graphics-aam/#mapping_role_table), or similar. This allows tools to present and support interaction with the object in a manner that is consistent with user expectations about other objects of that type.
```
Comment (EmmaJP) [inline pages/glossary/semantic-role.md]:
```suggestion
A _semantic [role](https://www.w3.org/TR/wai-aria-1.1/#dfn-role)_ is a semantic association that indicates an object's type. This allows tools to present and support interaction with the object in a manner that is consistent with user expectations about other objects of that type.

Valid semantic roles are defined by standards. For web content and applications these are defined within the various [WAI-ARIA specifications](https://act-rules.github.io/glossary/#wai-aria-specifications).
```

Comment (EmmaJP) [reply pages/glossary/semantic-role.md]: @Jym77 does that suggestion cover it ^^

Comment (Jym77) [reply pages/glossary/semantic-role.md]: Yes, I think it's good.

**Comment by EmmaJP** [comment]
Thanks @jkodu

**Pull request merged**